modules/tinymce/src/core/main/ts/api/Options.ts

```diff
@@ -859,6 +859,18 @@ const register = (editor: Editor): void => {
     default: true
   });
 
+  registerOption('selection_mode_allowed_commands', {
```

Comment:
This may not be a good idea 🤔 I was thinking since the ui's are all disabled in the upcoming PR for uiEnabled, if we might just want to remove this.

Reply:
Maybe in the future we can add an "allowed in readonly" property to commands as well

Reply:
This feels like an implementation detail we don't want to hard code into a setting just yet. Once a setting has been released it's hard to change.

I'm also a bit concerned that we have premium features listed here. I'd rather the premium feature remain in control of when the commands do and don't work.

Reply:
I've removed this for this version, maybe we can explore different opportunities in the upcoming version.